public/celestia-app.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+
+# ASCII art
+echo "          __        __  _                       "
+echo " _______ / /__ ___ / /_(_)__ ________ ____  ___ "
+echo "/ __/ -_) / -_|_-</ __/ / _ \`/___/ _ \`/ _ \/ _ \\"
+echo "\\__/\\__/_/\\__/___/\\__/_/\\_,_/    \\_,_/ .__/ .__/"
+echo "                                    /_/  /_/    "
+
+# Rest of your script...
+echo ""
```

Comment:
[not blocking] I didn't see this ASCII art

Reply:
same as above, newer version of script contains ascii